config/routes.rb

```diff
@@ -89,6 +89,7 @@
     get 'status', to: 'admin#status'
 
     get 'ppiu/payment_information', to: 'ppiu#index'
+    post 'ppiu/payment_information', to: 'ppiu#update'
```

Comment:
Should this verb be `patch` (or `put`)?
Are we creating (POST) a payment info object or are we updating (PATCH) the payment info of a person object.
If the verb stays POST, should we change the action to the typical `create` instead of `update` (typical for PATCH/PUT)?